modules/tinymce/src/plugins/table/main/ts/actions/Clipboard.ts

```diff
@@ -8,13 +8,10 @@
 import { Arr, Fun, Optional } from '@ephox/katamari';
 import { CopySelected, TableFill, TableLookup } from '@ephox/snooker';
 import { Replication, SugarElement, SugarElements, SugarNode } from '@ephox/sugar';
-
 import Editor from 'tinymce/core/api/Editor';
-
+import { SelectionTypes, Selections } from '@ephox/darwin';
```

Comment:
Another import out of order. External imports should be before internal imports.